Apps/frontend/src/app.scss

```diff
@@ -179,33 +190,47 @@ nav {
     }
 
     & > table {
+      display: flex;
+      flex-flow: column;
+      height: 100%;
       width: 100%;
-      text-align: center;
-      border-collapse: collapse;
-      table-layout: fixed;
-
-      & > th {
-        vertical-align: middle;
-        font-weight: normal;
-      }
-
-      & > * {
-        padding: 0;
-        margin: 0;
-        width: auto;
-
-        & > * {
-          padding: 0;
-          margin: 0;
-        }
-      }
+      border-collapse: unset;
+    }
+    & > table thead {
+      /* head takes the height it requires, 
+      and it's not scaled when table is resized */
+      flex: 0 0 auto;
+      width: calc(100% - 0.9em);
+    }
+    & > table tbody {
+      /* body takes all the remaining available space */
+      flex: 1 1 auto;
+      display: block;
+      overflow-y: scroll;
+    }
+    & > table tbody tr {
+        width: 100%;
+        text-align: center;
+    }
+    & > table thead, table tbody tr {
+        display: table;
+        table-layout: fixed;
+    }
+    & > tr {
+      padding: 3px;
+    }
+    & > table tr:nth-child(even) {
+      background-color: #838383;
+    }    
+    & > table tr:nth-child(odd) {
+      background-color: #999999;
     }
   }
 
   &--entry {
-    & > span {
-      position: absolute;
-    }
+    & > input {
```

Comment:
For what do we need `input` here? And why remove the `span: absolute` and then change it to an `inline div`?

Reply:
We don't have any spans inside the source code anymore. span caused a weird behavior so I changed it to div